### ISSUE TITLE
Exclude window classes from tiling

### ISSUE THREAD

**Issue by tflori** [ChWick/gnomesome#42]
I would need to have windows excluded from tiling. For example some applications look pretty ugly or don't allow to resize.

It would be easy to exclude them hardcoded. The problem for me is to write it so that you can have preferences for it. Or at least store it in dconf. Maybe you can give me a hint on that than I can prepare a PR if you are interested in such an extension.

Otherwise I will stay with my clone and hardcoded excludes. Here is the hardcoded example:
https://github.com/tflori/gnomesome/commit/ef110d683bd9e6d8cb6d43128d54bba90d8b2045

**Comment by tflori** [comment]
One more commit to allow tiling a floater: https://github.com/tflori/gnomesome/commit/61b8b212b885f12f0c1c95be4b841bcdd5568a08

**Comment by krillin666** [comment]
Any news on adding this as an option in the extension preferences ? Would be useful for applications which should not tile (e.g. ulauncher).

**Comment by ad1tyawagh** [comment]
Yes, I want this functionality for the exact purpose as @krillin666 . Any ETA about this?

**Comment by tflori** [comment]
FYI: I'm not using gnomesome anymore (switched to paperwm) and I will not write a PR for that.. I'll not remove the fork so if you are interested in it feel free to use these two commits...

that no maintainer is answering in 5 month speaks volumes.
